frontend/components/Header/Nav/index.js

```diff
@@ -55,58 +56,62 @@ export default class Nav extends Component<{}, { showMainMenu: boolean }> {
-                        role="navigation"
-                        aria-label="Guardian sections"
-                    >
-                        <EditionDropdown />
-                        <Logo />
-                        <Links />
-                        <Pillars
-                            showMainMenu={showMainMenu}
-                            pillars={nav.pillars || []}
-                        />
-                        <MainMenuToggle
-                            showMainMenu={showMainMenu}
-                            toggleMainMenu={toggleMainMenu}
-                            ariaControls={mainMenuId}
-                        />
-                        <MainMenu
-                            showMainMenu={showMainMenu}
-                            id={mainMenuId}
-                            nav={nav}
-                        />
-                    </nav>
-                    {nav.subNavSections &&
-                        nav.subNavSections.parent &&
-                        nav.subNavSections.links && (
-                            <div className={subnav}>
-                                <div className={centered}>
-                                    <SubNav
-                                        parent={nav.subNavSections.parent}
-                                        links={nav.subNavSections.links}
-                                    />
-                                </div>
+        // TODO push subnav into nav for now as really part of it
+        // also separate PR to do centering stuff!
+        return (
+            <div>
+                <nav
+                    className={centered}
+                    role="navigation"
+                    aria-label="Guardian sections"
+                >
+                    <EditionDropdown />
+                    <Logo />
+                    {/* 
+                        TODO: The properties of the Links component
+                        have been hardcoded to false. At some point 
+                        these need to be dynamic.
+                    */}
+                    <Links
+                        isPayingMember={false}
+                        isRecentContributor={false}
+                        isSignedIn={false}
+                    />
+                    <Pillars
+                        showMainMenu={showMainMenu}
+                        pillars={nav.pillars || []}
+                    />
+                    <MainMenuToggle
+                        showMainMenu={showMainMenu}
+                        toggleMainMenu={toggleMainMenu}
+                        ariaControls={mainMenuId}
+                    />
+                    <MainMenu
+                        showMainMenu={showMainMenu}
+                        id={mainMenuId}
+                        nav={nav}
+                    />
+                </nav>
+                {nav.subNavSections &&
+                    nav.subNavSections.parent &&
+                    nav.subNavSections.links && (
```

Comment:
This is not really part of this PR, but we should pull out logic like this into named variables. E.g.

    const hasSubnav = nav.subNavSections && ..parent && ..links

(I think I wrote the original code so note for future Nic.)